services/QuillLMS/spec/helpers/navigation_helper_spec.rb

```diff
@@ -196,4 +160,229 @@
       expect(helper.should_show_admin_access_tab?).to eq true
     end
   end
+
+  describe '#determine_mobile_navbar_tabs' do
```

Comment:
I love how extensive and thorough these tests are.

Reply:
Agreed with Thomas. These tests do, however, make me feel like using a hash structure like the one Thomas was suggesting might be more maintainable - because then you could just iterate through that to test that things return as expected rather than manually writing out each path in both places.